### DISCUSSION TITLE
NOSTROS login using pub key vs pri key?

### DISCUSSION THREAD

**Discussion by jairunet** [KoalaSat/nostros#380] [Q&A] [answered]
Hello, NOSTROS Community,

Thank you for your hard work, I am curious about the limitations I will get by login in with my public key instead of the private one.  As we all understand the private key should be kept safe.  If I post my private key or reveal it online I believe this will be a security issue.

Can you please briefly explain the disadvantages of login into NOSTROS using just the public key if there are any?

I appreciate all you do.

Thank you again!

**Comment by KoalaSat** [comment, marked as answer]
Login with a public key is like using a "Read only" mode. You cannot post, decrypt DM or do anything that will required a signature with your private key. 

The point of this mode is also that you can login with virtually any `pubkey` and see how other users see nostr.

**Reply by jairunet** [reply]
Make sense,

Thank you very much for the prompt and clear response.

Sincerely,
